File: Practica2/circulo.py
from punto import Punto

class Circulo:
    def __init__(self, centro: Punto, radio: float):
        self.centro = centro
        self.radio = radio
    
    def __str__(self):
        return f"Circulo(centro={self.centro}, radio={self.radio})"
    
    def dibujaCirculo(self):
        print(f"Dibujando círculo con centro {self.centro} y radio {self.radio}")

# Ejemplo de uso:
if __name__ == "__main__":
    c = Circulo(Punto(2, 2), 5)
    print(c)
    c.dibujaCirculo()

File: Practica2/circulo.java
public class Circulo {
    private Punto centro;
    private float radio; 

    public Circulo(Punto centro, float radio) {
        this.centro = centro;
        this.radio = radio;
    }

    @Override
    public String toString() {
        return "Circulo(centro=" + centro + ", radio=" + radio + ")";
    }

    public void dibujaCirculo() {
        System.out.println("Dibujando círculo con centro " + centro + " y radio " + radio);
    }

    // Ejemplo de uso en un main (opcional)
    public static void main(String[] args) {
        Punto c = new Punto(2, 2);
        Circulo circulo = new Circulo(c, 5.0f);
        System.out.println(circulo);
        circulo.dibujaCirculo();
    }
}
